visual_behavior/ophys/response_analysis/response_processing.py

```diff
@@ -413,9 +413,9 @@ def response_df(response_xr):
     return df
 
 
-def filter_events_array(trace_arr, scale=2):
+def filter_events_array(trace_arr, t_scale, scale=2):
     from scipy import stats
-    filt = stats.halfnorm(loc=0, scale=scale).pdf(np.arange(20))
+    filt = stats.halfnorm(loc=0, scale=scale).pdf(np.arange(t_scale))
```

Comment:
I'm not sure t_scale is the problem. t_scale will make the filter vector longer, but not change the width of the gaussian. So it will just add more zeros to the end. 

The problem is the `scale` parameter which determines the width of the gaussian.